src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/ShortcutGuidePage.xaml

```diff
@@ -29,12 +32,36 @@
 
                 <controls:SettingsGroup x:Uid="Shortcut" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
 
-                    <controls:Setting x:Uid="Activation_Shortcut" Icon="&#xEDA7;" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
+                    <controls:Setting x:Uid="ShortcutGuide_ActivationMethod">
+                        <controls:Setting.ActionContent>
+                            <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.UseLegacyPressWinKeyBehavior, Converter={StaticResource BoolToComboBoxIndexConverter}}" MinWidth="{StaticResource SettingActionControlMinWidth}">
+                                <ComboBoxItem x:Uid="Radio_ShortcutGuide_ActivationMethod_CustomizedShortcut"/>
+                                <ComboBoxItem x:Uid="Radio_ShortcutGuide_ActivationMethod_LongPressWindowsKey"/>
+                            </ComboBox>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
+                    <controls:Setting x:Uid="Activation_Shortcut" Icon="&#xEDA7;" Visibility="{x:Bind Mode=OneWay, Path=ViewModel.UseLegacyPressWinKeyBehavior, Converter={StaticResource FalseToVisibleConverter}}">
                         <controls:Setting.ActionContent>
                             <controls:ShortcutControl HotkeySettings="{x:Bind Path=ViewModel.OpenShortcutGuide, Mode=TwoWay}"
                                                             MinWidth="{StaticResource SettingActionControlMinWidth}"/>
                         </controls:Setting.ActionContent>
                     </controls:Setting>
+
+                    <controls:Setting x:Uid="ShortcutGuide_PressTime" Icon="&#xF0E2;" Visibility="{x:Bind Mode=OneWay, Path=ViewModel.UseLegacyPressWinKeyBehavior, Converter={StaticResource TrueToVisibleConverter}}">
+                        <controls:Setting.ActionContent>
+                            <muxc:NumberBox
+                                Minimum="100"
+                                Value="{x:Bind Mode=TwoWay, Path=ViewModel.PressTime}"
+                                MinWidth="{StaticResource SettingActionControlMinWidth}"
+                                SpinButtonPlacementMode="Compact"
+                                HorizontalAlignment="Left"
+                                SmallChange="50"
+                                LargeChange="100"
+                            />
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
```

Comment:
Not sure what was concluded on the topic of showing a warning message when using the long-press functionality. If we do want to do this, you can use something like this:

<muxc:InfoBar Severity="Warning" Message="In some edge cases Shortcut Guide might not function correctly when using this method" IsOpen="{x:Bind Mode=OneWay, 
 Path=ViewModel.UseLegacyPressWinKeyBehavior}" IsCloseable="False" />

Reply:
Thank you. I'll add that in.

Reply:
![image](https://user-images.githubusercontent.com/26118718/134425454-2f7a018f-812b-4081-a837-355e1ea0527d.png)